### ISSUE TITLE
No "Manage Private Keys" option after importing certificate

### ISSUE THREAD

**Issue by DanSibbernsen** [chef-cookbooks/windows#575]
### Cookbook version
5.1.0

### Chef-client version
14.2.0

### Platform Details
Windows Server 2016 - latest updates
### Scenario:
I am trying to use a certificate after it has been installed on my machine. When I run mmc.exe, load the 'Certificates' snap-in, select the "Computer Account" and right-click on the certificate, there is no "Manage Private Keys" under "All Tasks." When I push the same certificate manually through mmc, I do have that option.

### Steps to Reproduce:
command:
```
kitchen converge
```
recipe:
```
passwords = data_bag_item("certificates", "passwords")
password = passwords['generic_field']

windows_certificate "/path/to/certificate.pfx" do
  pfx_password password
  private_key_acl ['Administrators']
  store_name 'MY'
  action :create
end
```

### Expected Result:
I expect the certificate to be imported and have the "Manage Private Keys" as an option.
### Actual Result:
No "Manage Private Keys" option is available.

### Additional Notes
From some general google searching it looks like this might be fixed by specifying the "-sky" option for makecert.exe. I'm not sure if we're using this executable somewhere under the hood, but that's where I would start an investigation.

**Comment by lewisgaines** [comment]
Commenting for visibility. This issue started with the change to start using the win32_certificate ruby gem in version 4.3.0. Another related issue with a bit mroe detail is https://github.com/chef-cookbooks/windows/issues/580.

**Comment by tas50** [comment]
This will be resolved in the new release I'm about to push and is also fixed in Chef 14.11
